scripts/extensions/datetimeField/src/getTemplateDateTimeField.tsx

```diff
@@ -14,7 +14,7 @@ interface IState {
 export function getTemplateDateTimeField(superdesk: ISuperdesk) {
     const {Spacer} = superdesk.components;
     const currentDateTime = '{{ now|iso_datetime }}';
-    const DateTimeField: any = getDateTimeField(superdesk);
+    const DateTimeField = getDateTimeField(superdesk);
```

Comment:
Why did you set it to `any`? Can you go over this extension and check that `any` is not used where not really needed?

I was searching for usages of a prop using IDE, and it found none because of this.